### PR TITLE
[Communication] -Administration- changing some createReservation from public to private

### PR DESCRIPTION
**Pull request by paolamvhz**

### CODE REVIEW

File: sdk/communication/azure-communication-administration/README.md [hunk 1]
```diff
@@ -39,47 +39,47 @@ via the endpoint() and httpClient() functions respectively.
 
 <!-- embedme ./src/samples/java/com/azure/communication/administration/ReadmeSamples.java#L37-L48 -->
 ```java
-// You can find your endpoint and access key from your resource in the Azure Portal
```
Comment (ankitarorabit) [inline sdk/communication/azure-communication-administration/README.md:42]: Not sure why this was removed from the samples?

File: sdk/communication/azure-communication-administration/README.md [hunk 2]
```diff
@@ -39,47 +39,47 @@ via the endpoint() and httpClient() functions respectively.
...
+
+
+
+
 ```
 
 Alternatively, you can provide the entire connection string using the connectionString() function instead of providing the endpoint and access key. 
 <!-- embedme ./src/samples/java/com/azure/communication/administration/ReadmeSamples.java#L62-L68 -->
 ```java
-// Your can find your connection string from your resource in the Azure Portal
```
Comment (ankitarorabit) [inline sdk/communication/azure-communication-administration/README.md:59]: Same comment as above

File: sdk/communication/azure-communication-administration/README.md [hunk 3]
```diff
@@ -118,44 +118,44 @@ also takes in a list of communication token scopes. Scope options include:
...
 System.out.println("Expires On: " + userToken.getExpiresOn());
+return userToken;
 ```
 
 ### Revoking all tokens for an existing user
 Use the `revokeTokens` function to revoke all the issued tokens of a user.
 
 <!-- embedme ./src/samples/java/com/azure/communication/administration/ReadmeSamples.java#L108-L109 -->
 ```java
-// revoke tokens issued for the user prior to now
```
Comment (ankitarorabit) [inline sdk/communication/azure-communication-administration/README.md:132]: Why this change? The existing snippet seemed fine

**Review by ankitarorabit — CHANGES_REQUESTED** [review]
Readme has a lot of changes that don't seem relevant. Could you please double check and remove the unintended changes

File: sdk/communication/azure-communication-administration/README.md (hunk 1 above)
Comment (paolamvhz) [reply sdk/communication/azure-communication-administration/README.md]: Fixed there was an unsed impor removed from the sample files that broke almost everything

File: sdk/communication/azure-communication-administration/README.md (hunk 2 above)
Comment (paolamvhz) [reply sdk/communication/azure-communication-administration/README.md]: Same as the issue above

File: sdk/communication/azure-communication-administration/README.md (hunk 3 above)
Comment (paolamvhz) [reply sdk/communication/azure-communication-administration/README.md]: same as the issue above

**Review by ankitarorabit — APPROVED** [review]
